def/de/settings.cson

```diff
@@ -138,51 +138,36 @@ Settings:
       }
       {
         _id: 'core.projectHome'
-        title: "Projekt Heimverzeichnis"
-        desc: "Verzeichnis in dem Projekte gespeichert werden. Pakete welche mittels dem Paket-Generator erstellt werden, werden dort standardmäßig abgelegt."
+        title: "Projekt-Heimverzeichnis"
+        desc: "Verzeichnis, in dem Projekte gespeichert werden. Pakete, welche mittels dem Packages-Generator erstellt werden, werden dort standardmäßig abgelegt."
       }
       {
         _id: 'core.reopenProjectMenuCount'
-        title: "Anzahl kürzlich geöffneter Projekte im Projekt Menü"
-        desc: "Wie viele Projekte welche im Menü der kürzlich geöffneten Projekte angezeigt werden."
+        title: "Anzahl kürzlich geöffneter Projekte im Projekt-Menü"
+        desc: "Anzahl der Projekte, die im Menü &quot;Kürzlich geöffnete Projekte&quot; angezeigt werden sollen."
       }
       {
         _id: 'core.restorePreviousWindowsOnStart'
         title: "Letzte Fenster beim Start wiederherstellen"
-        desc: "Wenn aktiviert, wird der letzte Stand aller Atom-Fenster bei Start über das Icon wiederhergestellt. Gilt auch für den Start von <code>atom</code> über die Kommandozeile. Andernfalls wird Atom mit einer neuen Umgebung gestartet."
+        desc: "Sofern aktiviert, wird der letzte Stand aller Atom-Fenster bei Start über das Icon wiederhergestellt. Gilt auch für den Start von <code>atom</code> über die Kommandozeile. Andernfalls wird Atom mit einer neuen Umgebung gestartet."
         select:
           no: "Nein"
           yes: "Ja"
           always: "Immer"
       }
       {
         _id: 'core.telemetryConsent'
-        title: "Sende Telemetrie-Daten an das Atom Team"
-        desc: "Dem Senden und der Verwendung von statistischen Daten sowie Ausnahmemeldungen an das Atom Team zustimmen. Dient der Produkt um mitzuhelfen das Produkt zu verbessern."
+        title: "Sende Telemetrie-Daten an das Atom-Team"
+        desc: "Dem Senden und der Verwendung von statistischen Daten sowie Ausnahmemeldungen an das Atom-Team zustimmen. Dies hilft uns, um Atom noch besser zu machen."
         select:
           limited: "Limitierte anonyme Nutzer-Statistiken, Ausnahmen und Fehlerberichte"
           no: "Keine Telemetrie-Daten senden"
-          undecided: "Unentschlossen (Atom fragt beim nächsten Start erneut nach)"
-      }
-      {
-        _id: 'core.titleBar'
```

Comment:
Please keep this section `core.titleBar` to keep consistency of `settings.cson` of all locales.
this part may appear in older versions.